gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/base.py.j2

```diff
@@ -183,6 +183,14 @@ class {{ service.name }}Transport(abc.ABC):
             {% endfor %} {# precomputed wrappers loop #}
         }
 
+    def close(self):
+        """Closes resources associated with the transport.
+
+       :: warning::
```

Comment:
I think this should have the same syntax as the other warning infoboxes, right?

```suggestion
       .. warning::
```

Reply:
Which one are you talking about? 

I am following syntax from the [docs](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html).

Reply:
Per the docs you linked, Restructured Text  directives like `note`, `warning`, `code-block` all start with two periods, then a space, then the directive name, then two colons.

Reply:
Ah, thank you. I see now.